### PR TITLE
Update codeowners for laa aws infra

### PR DESCRIPTION
**Pull request by vc13837**
## What

[Link to story](https://dsdmoj.atlassian.net/browse/AP-XXX)

Describe what you did and why.

## Checklist

Before you ask people to review this PR:

- [ ] Tests should be passing: `./gradlew test`
- [ ] Github should not be reporting conflicts; you should have recently run `git rebase main`.
- [ ] Avoid mixing whitespace changes with code changes in the same commit. These make diffs harder to read and conflicts more likely.
- [ ] You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- [ ] You should have checked that the commit messages say why the change was made.

### CODE REVIEW

**Review by stone-dj — COMMENTED** [review]
Would any of these files also be of interest from an Ops perspective, if so, can you include them also?

- [Dockerfile](https://github.com/ministryofjustice/laa-maat-court-data-api/blob/fb75820877295903bfd630ca5b56027e3e5d870b/maat-court-data-api/Dockerfile)
- [maat-cd-api-deployment-pipeline.template](https://github.com/ministryofjustice/laa-maat-court-data-api/blob/8c5275aabbd97243c156e354f7522abef12028cb/aws/pipeline/maat-cd-api-deployment-pipeline.template)
- [.git-crypt/.gitattributes](https://github.com/ministryofjustice/laa-maat-court-data-api/blob/6b35eeb05682af8946ef604f4b6e32dcf69c0fb4/.git-crypt/.gitattributes)
- [.github/CODEOWNERS](https://github.com/ministryofjustice/laa-maat-court-data-api/blob/a6f053d06efc9fe2dbdace3517651c47f317ffe1/.github/CODEOWNERS)

**Comment by vc13837** [comment]
Hi Alex, thanks for the suggestions. As Ops we will need the deployment pipeline template, but that should be covered by the `/aws/` part of the permissions. As for the rest, I think the image building and repo ownership would make sense to get Crime Apps' approval should we need to make changes.

**Review by stone-dj — APPROVED** [review]
Changes look good, comment addressed

**Pull request merged**
